meta-refkit-core/lib/oeqa/selftest/cases/refkit_license_check.py

```diff
@@ -88,15 +92,17 @@ def _get_latest_manifest(self, imagename, deploydir):
 
         return os.path.join(manifestdir, "package.manifest")
 
-    def test_check_computervision_licensing(self):
+    def test_check_computervision_licensing(self, test_image):
```

Comment:
I'd suggest you add a default value for test_image here. With that, the printing below can be simplified to 

```python
print("testing: %s" % test_image
```